assets/js/stripe-payment-request.js

```diff
@@ -548,7 +548,12 @@ jQuery( function( $ ) {
 				});
 			} );
 
-			$( '.quantity' ).on( 'change', '.qty', wc_stripe_payment_request.debounce( 250, function() {
+			// To avoid an error when a user clicks the button before it updates.
```

Comment:
Could we add some details to this comment? Maybe something like:
```suggestion
			// Block the payment request button as soon as an "input" event is fired, to avoid sync issues
			// when the customer clicks on the button before the debounced event is processed.
```